vapi/rest/client.go

```diff
@@ -125,6 +125,17 @@ func (c *Client) Resource(path string) *Resource {
 	return r
 }
 
+// Resource helper for the given path.
+// Don't use this one unless you have a particular reason to
+func (c *Client) ResourceWithRawPath(path string) *Resource {
```

Comment:
Seems to be unused, so any reason this is part of the PR?

nit: doc string is invalid

Reply:
Oops, that's just a failed experiment, will remove